src/main/scala/mesosphere/marathon/api/akkahttp/v2/PodsController.scala

```diff
@@ -0,0 +1,113 @@
+package mesosphere.marathon
+package api.akkahttp.v2
+
+import akka.http.scaladsl.model.Uri.Path
+import akka.http.scaladsl.server.Route
+import mesosphere.marathon.api.akkahttp.Controller
+import mesosphere.marathon.api.akkahttp.Directives.extractInstanceId
+import mesosphere.marathon.api.akkahttp.PathMatchers.{ ExistingRunSpecId, PodsPathIdLike }
+import mesosphere.marathon.core.group.GroupManager
+import mesosphere.marathon.core.instance.Instance
+import mesosphere.marathon.plugin.auth.Authenticator
+import mesosphere.marathon.state.PathId
+import akka.http.scaladsl.server.PathMatchers
+
+class PodsController(
+    val groupManager: GroupManager)(
+    implicit
+    val authenticator: Authenticator) extends Controller {
+
+  import mesosphere.marathon.api.akkahttp.Directives._
+
+  def capability(): Route =
+    authenticated.apply { implicit identity =>
+      complete("")
```

Comment:
You can also use `complete(StatusCodes.OK)`

Reply:
Good point.